packages/colors/src/colors.js

```diff
@@ -268,6 +268,6 @@ export const colors = {
     0: white,
   },
   yellow: {
-    20: yellow20,
+    30: yellow30,
```

Comment:
I think we'll also need to keep yellow20 in here, as well, technically

Reply:
Should I leave yellow as yellow 30 and add yellow 20 as its own ?

Reply:
I think adding them both under yellow is fine 👍 This is to prevent code from breaking when it uses `colors.yellow['20']` 👍

Reply:
Done